### ISSUE TITLE
Placing Nuclear Boiler/Particle Accelerator

### ISSUE THREAD

**Issue by Breadabix** [halvors/Quantum#59]
When placing machines they place facing towards you, except the nuclear boiler and the particle accelerator block which place facing away from you. 
This is in 1.7.10 with latest version of this mod and forge

**Comment by halvors** [comment]
@Breadabix Pretty sure nuclear boiler is supposed to face that way? Any references on how that was in the original Atomic Science?

**Comment by Breadabix** [comment]
In voltz it places the other way to what it does in this.

**Comment by halvors** [comment]
@Breadabix Yes, but if you look at the machine, controls is on the back of it. Or is that just me?

**Comment by Breadabix** [comment]
This is how it places in voltz, controls and the front and the back the things that spin.
https://gyazo.com/56233ba4cf80fcce97796e39ebf81db6
